php/PhpJsonApiSamples/ClientSideEncryptionDemo2.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<!--
+To change this license header, choose License Headers in Project Properties.
+To change this template file, choose Tools | Templates
+and open the template in the editor.
+-->
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Client side Encryption</title>
+         <!-- This is the PayTrace End-to-End Encryption library: -->
+        <script src="https://api.paytrace.com/assets/e2ee/paytrace-e2ee.js"></script>
+         <!-- This page shows Demo for using PayTrace client side Encryption library to submit the form without checking any form validations.-->
+       
+    </head>
+    <body>
+        <script>
+            // this hooks form submission.
```

Comment:
Not sure if this explanation in this comment is to vague. Maybe this?
Code example of hooking up the form submit event to the PayTrace encryption library.